python/tvm/meta_schedule/tune.py

```diff
@@ -407,6 +424,15 @@ def tune_tir(
         params=[{"log_dir": log_dir, "logger_name": __name__ + f".task_{task_name}"}],
     )
 
+    if blocks is not None:
+        assert space is None, "Only one of blocks and space can be specified."
```

Comment:
Nit, we can be more specific with the warning here `Can not specify blocks to tune when search space is given.`